### ISSUE TITLE
Exception in background thread killing w3wp

### ISSUE THREAD

**Issue by zpqrtbnk** [Shazwazza/Examine#47]
Seeing this exception in Umbraco v8 when displaying the Examine dashboard.

```
   Lucene.Net.Store.AlreadyClosedException
   à Lucene.Net.Index.IndexReader.EnsureOpen()
   à Lucene.Net.Index.IndexReader.IncRef()
   à Examine.LuceneEngine.Cru.SearcherManager.AcquireSearcher()
   à Examine.LuceneEngine.Cru.SearcherManager.get_IsSearcherCurrent()
   à Examine.LuceneEngine.Cru.NrtManager.MaybeReopen(Boolean)
   à Examine.LuceneEngine.Cru.NrtManagerReopener.Start()
   à System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   à System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   à System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   à System.Threading.ThreadHelper.ThreadStart()
```

No idea why the thing is throwing but, throwing in a background thread kills w3wp entirely. Creating the issue to keep a reference & details.

**Comment by chernenkoartyom** [comment]
I've got the same issue in Umbraco 7.7.3 - w3wp was killed in a background and JIT debugger shown this

![image](https://user-images.githubusercontent.com/36659102/36414328-b9ff2db2-1643-11e8-8b66-c6a013f06c3b.png)

**Comment by Shazwazza** [comment]
@chernenkoartyom Any issue you are having in 7.7.3 will be very different from the one reported for the version 8 WIP branch.

You'll need to provide steps to replicate and more details on how to debug this.

Thanks!

**Comment by Shazwazza** [comment]
Closing as the issue was resolved in 1.0
